workspaces/repo-tools/.changeset/red-bananas-join.md

```diff
@@ -0,0 +1,5 @@
+---
+'@backstage-community/cli': patch
+---
+
+Removed @microsoft/api-extractor from resolutions in the `package.json` file as it is no longer needed since the Backstage `1.32.0` release
```

Comment:
let's remove this changeset as this plugin isn't published and we are good!

Reply:
Sure, done!